api/src/utils/env.ts

```diff
@@ -44,8 +46,7 @@ If so, ensure that the environment variable JEST_WORKER_ID is set.`
 assert.ok(process.env.HOME_LOCATION);
 assert.ok(process.env.FREECODECAMP_NODE_ENV);
 assert.ok(isAllowedEnv(process.env.FREECODECAMP_NODE_ENV));
```

Comment:
I would have thought we wanted to default:
- `FREECODECAMP_NODE_ENV`
- `HOME_LOCATION`
- `API_LOCATION`
- `SENTRY_ENVIRONMENT`
- `FCC_ENABLE_DEV_LOGIN`
- `COOKIE_DOMAIN`

No?

Reply:
Good question. Given that we're trying to cut down on deployment envs, I reckon we could default `FREECODECAMP_NODE_ENV` and `FCC_ENABLE_DEV_LOGIN`.

The rest we don't gain much by defaulting  since they differ between dev and org.